README.md

```diff
@@ -1 +1,42 @@
-# .github
\ No newline at end of file
+# About Us 
+
+[The Childhood Cancer Data Lab](https://www.ccdatalab.org/) was established by Alex’s Lemonade Stand Foundation (ALSF) in 2017. The Data Lab is a team of data scientists, designers, engineers, and communicators. Our mission is to accelerate the pace of finding novel cures and treatments for childhood cancer by putting resources and knowledge in the hands of pediatric cancer experts. 
+
+We construct tools that make vast amounts of data widely available, easily mineable, and broadly reusable. We also train researchers to better understand their own data and to advance their work more quickly. The Data Lab team simultaneously contributes to childhood cancer research and to the open science and open source software communities.
+
+# Our Projects
+[`refine.bio`](https://www.refine.bio/) is a multi-organism collection of genome-wide transcriptome or gene expression data that has been obtained from publicly available repositories and uniformly processed and normalized.
+ 
+* [Read the documentation.](http://docs.refine.bio/en/latest/)
+* [Get started](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html) with [example workflows for use with refine.bio data.](https://github.com/AlexsLemonade/refinebio-examples)
+* [Learn about building and running the refine.bio project source code](https://github.com/AlexsLemonade/refinebio).
+
+`The Single-cell Pediatric Cancer Atlas (ScPCA)` focuses on single-cell profiling with the goal of creating a publicly available atlas of single-cell pediatric cancer data. Ten ScPCA awards were funded by Alex’s Lemonade Stand Foundation. The ALSF funded researchers submit their single-cell, single-nuclei, and bulk RNA sequencing data to the Data Lab for processing. The data from these patient tumors will be made openly available in one location through the ScPCA Portal, which is currently under development. 
+
+* [Management and analysis tools for ScPCA data.](https://github.com/AlexsLemonade/alsf-scpca)
```

Comment:
Hm maybe link to `scpca-nf` instead? Tagging @jashapiro and @allyhawkins to weigh in

Reply:
Hmm yea I would either link to `scpca-nf` and/or `scpcaTools` as `alsf-scpca` contains more of the benchmarking analysis and the other two tools contain the pipelines we are using to actually do the final processing and analysis for the portal.

Reply:
`scpcaTools` isin the bullet point below this, so I think that's a vote for `scpca-nf` here!

Reply:
Thanks **@allyhawkins** and **@jaclyn-taroni**! Should the wording `Management and analysis tools for ScPCA data` be changed to something else as well, or does that still make sense?

Reply:
To better reflect what's actually in that repo, I might change it to be `Processing workflows for ScPCA data`.